_posts/2014-12-19-becoming-a-sponsor.md

```diff
@@ -5,35 +5,34 @@ title:  "Sponsoring"
 
 ### How to sponsor a workshop
 
-- Workshops usually range from 40-60 attendees, but can be higher or lower based on the capacity of the venue and the size of the chapter.
+- Depending on the location workshop attendee numbers can range from as low as 10 to as high as 60, so for exact details you should send an email to hello@codebar.io enquiring about the specifics of a chapter.
 
-The minimum requirements for hosting a workshop are desk space, power outlets, and a stable internet connection.
+- The minimum requirements for hosting a workshop are desk space, power outlets, and a stable internet connection
 
-Additionally, we encourage sponsors to provide food and drinks. Since codebar workshops typically take place immediately after work for many students and coaches, it is recommended that hosts provide some hot food, as attendees will not have another chance to eat dinner. In London, the typical spend for hot food is roughly £200-250 for one workshop; outside of central London it can be lower.
+- We encourage providing food and drinks. Since our workshops typically take place immediately after work for many students and coaches, it is recommended that hosts provide some hot food, as attendees will not have another chance to eat dinner. In London, the typical spend for hot food is roughly £200-250 for one workshop; outside of central London it can be much lower.
 
-With food, please provide vegetarian, vegan and gluten-free options, and with drinks, please provide non-alcoholic drinks if you are planning on providing beer or wine. If you would like recommendations for food, please get in touch with your chapter organizers.
+  - Food options: try to provide some vegetarian, vegan and gluten-free options - everyone will eat vegetarian food and not the other way around so try to keep it at a 50-50
+  - Drinks: Please make sure you provide non-alcoholic drinks and water. Some locations also prefer to keep their workshop alcohol free so make sure you speak with the organisers  if you are planning on providing beer or wine.
 
-We also like to hold lightning talks at the start of each event. If your space has a larger conference room with a projector (preferably HDMI and/or with an adapter for Mac Thunderports) please let us know in advance and have it ready to go on the day of the workshop.
+  If you would like recommendations for food, please get in touch with your chapter organizers.
```

Comment:
shouldn't this food line go above, under the food bit before we start talking about drinks?